libkbfs/disk_block_cache.go

```diff
@@ -138,17 +181,25 @@ func newDiskBlockCacheStandardFromStorage(config diskBlockCacheConfig,
 	startedCh := make(chan struct{})
 	startErrCh := make(chan struct{})
 	cache = &DiskBlockCacheStandard{
-		config:     config,
-		maxBlockID: maxBlockID.Bytes(),
-		tlfCounts:  map[tlf.ID]int{},
-		tlfSizes:   map[tlf.ID]uint64{},
-		log:        log,
-		blockDb:    blockDb,
-		metaDb:     metaDb,
-		tlfDb:      tlfDb,
-		startedCh:  startedCh,
-		startErrCh: startErrCh,
-		shutdownCh: make(chan struct{}),
+		config:           config,
+		maxBlockID:       maxBlockID.Bytes(),
+		tlfCounts:        map[tlf.ID]int{},
+		tlfSizes:         map[tlf.ID]uint64{},
+		hitMeter:         metrics.NewMeter(),
```

Comment:
Other parts of `libkbfs` (in `init.go` specifically) avoids creating meters when `config.MetricsRegistry() == nil`. Is there a good reason to assume the registry will always be non-nil here?  (I think there's an env variable or something that can turn it off.)